module-2/backend/week_one.md

```diff
@@ -18,15 +27,38 @@ Note: When you're done, submit a PR.
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+ - between do and end add @name = 'Mr. Ed'
 9. What's the purpose of ERB?
+  - to embed ruby in html
 10. Why do I need a development AND test database?
+  - the test database needs to be torn down every time. you may want to hold data in the development database.
 11. What's responsive design?
+  - adjusting page layout to different resolutions and sizes
 12. What is CRUD and why is it important?
+  - create, read, update, destroy. the 4 main things we want to do to databases.
 13. What does HTTP stand for? 
+  - Hypertext transfer protocol
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+  - <% ruby %> wont appear on page. <%= ruby %> will appear on page
 15. What's an ORM?
+  - object relational mapper. allows us to treat database rows as objects.
 16. What's the most commonly used ORM?
+  - active record
 17. Let's say we have an application with restaurants. There are seven verb + path combinations necessary to provide full CRUD functionality for our restaurant application. List each of the seven combinations, and explain what each is for.
+  - CREATE
+    - GET /new - view new record template

```

Comment:
if you're using "thing" as your noun here... then we'd have the following:
GET '/things/new' - view new record template for a thing
POST '/things' - create the new thing
GET '/things' - view all the things
GET '/things/:id' - view specific thing
GET '/things/:id/edit - view edit template for thing
PUT '/things/:id' - update thing
DELETE '/things/:id' - delete thing

Notice that `things` is always plural and we always have it in the pathname 👍